Checkout/Checkout/Source/Model/Card+Scheme.swift

```diff
@@ -29,13 +31,13 @@ extension Card {
     case mada
     case visa
     case mastercard
-    case maestro
+    case maestro(length: Int = 0)
     case americanExpress
     case discover
     case dinersClub
     case jcb
 
-    public init?(rawValue: String) {
+    public init?(rawValue: String, length: Int? = nil) {
       switch rawValue.lowercased() {
```

Comment:
`Scheme` rawValue is no longer String. so why need for init with string rawValue?

Reply:
The only reason its no longer having a rawValue of String is the limitation on enums with associated values. Their conformance cannot be inherited.

We do however still need the behaviour for encoding, decoding and lots of parsing happening, which is why we do need to enable behaviour ourselves.